mesos-client/src/main/scala/com/mesosphere/mesos/client/MesosClient.scala

```diff
@@ -150,52 +154,52 @@ object MesosClient extends StrictLogging with StrictLoggingFlow {
   private def connectionSource(frameworkInfo: FrameworkInfo, url: URI)(implicit as: ActorSystem) = {
     val body = newSubscribeCall(frameworkInfo).toByteArray
 
-    val request = HttpRequest(
-      HttpMethods.POST,
-      uri = Uri("/api/v1/scheduler"),
-      entity = HttpEntity(ProtobufMediaType, body),
-      headers = List(headers.Accept(ProtobufMediaType)))
+    val request = HttpRequest(HttpMethods.POST,
+                              uri = Uri("/api/v1/scheduler"),
+                              entity = HttpEntity(ProtobufMediaType, body),
+                              headers = List(headers.Accept(ProtobufMediaType)))
```

Comment:
Shouldn't this be just and indent as well like on the left?

Reply:
But I don't like it. I mean we can discuss or just use what we had for Marathon.

Reply:
are you saying it's harder to read (or what's 'not liking it?'). Because to me it's definitely the opposite. It would be harder to write, but you don't have to write it...

Reply:
this is one of the formats that I get frustrated with regarding python and pep.   The alignment to the open brace along with line length limits leads to frustrating IMO.   I'm with karsten...  previous indicate is preferred.   it is clear that it is a continuation of the function declaration and allows for verbosity where needed.